frontend/graph/schema.graphqls

```diff
@@ -91,14 +75,13 @@ input K8sNamespaceId {
 
 type K8sActualSource {
   namespace: String!
-  kind: K8sResourceKind!
   name: String!
-  serviceName: String
+  kind: K8sResourceKind!
   numberOfInstances: Int
+  serviceName: String
```

Comment:
same here

Reply:
ServiceName is the name with "kind" as prefix (eg. deployment-load-generator).
We don't use it in the UI, but it could be nice to have 100% of the data